{"category": "PytestAssertRewriteWarning", "filename": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", "lineno": 1204, "message": "Module already imported so cannot be rewritten: anyio", "$report_type": "WarningMessage", "when": "config", "location": ["/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", 1204, "_consider_importhook"]}
{"category": "PytestAssertRewriteWarning", "filename": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", "lineno": 1204, "message": "Module already imported so cannot be rewritten: anyio", "$report_type": "WarningMessage", "when": "config", "location": ["/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", 1204, "_consider_importhook"]}
{"pytest_version": "7.4.4", "$report_type": "SessionStart"}
{"nodeid": "tests58_52_2.py::test_horizontal_matrix", "location": ["tests58_52_2.py", 40, "test_horizontal_matrix"], "keywords": {"test_horizontal_matrix": 1, "__wrapped__": 1, "tests58_52_2.py": 1, "Folder_52": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.0001421910710632801, "start": 1731152876.1244202, "stop": 1731152876.1245627, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw2", "testrun_uid": "3d3ace11be9e4282adfa52696af4d5e4", "node": "<WorkerController gw2>"}
{"nodeid": "tests58_52_2.py::test_matrix_of_one_integer", "location": ["tests58_52_2.py", 18, "test_matrix_of_one_integer"], "keywords": {"test_matrix_of_one_integer": 1, "__wrapped__": 1, "tests58_52_2.py": 1, "Folder_52": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.0001410529948771, "start": 1731152876.1263003, "stop": 1731152876.126442, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "3d3ace11be9e4282adfa52696af4d5e4", "node": "<WorkerController gw0>"}
{"nodeid": "tests58_52_2.py::test_matrix_of_having_147_integers", "location": ["tests58_52_2.py", 28, "test_matrix_of_having_147_integers"], "keywords": {"test_matrix_of_having_147_integers": 1, "__wrapped__": 1, "tests58_52_2.py": 1, "Folder_52": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00014886283315718174, "start": 1731152876.1302638, "stop": 1731152876.130413, "$report_type": "TestReport", "item_index": 1, "worker_id": "gw1", "testrun_uid": "3d3ace11be9e4282adfa52696af4d5e4", "node": "<WorkerController gw1>"}
{"nodeid": "tests58_52_2.py::test_horizontal_matrix", "location": ["tests58_52_2.py", 40, "test_horizontal_matrix"], "keywords": {"test_horizontal_matrix": 1, "__wrapped__": 1, "tests58_52_2.py": 1, "Folder_52": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "numpy.exceptions.AxisError: axis 2 is out of bounds for array of dimension 2"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_horizontal_matrix():", "        mat = [[random.randint(-10, 10) for _ in range(147)]]", "        mat = np.asmatrix(mat)", "        m = np.concatenate((mat, mat), axis=1)", "        if 147 < 30:", "            num = 4", "        elif 30 <= 147 < 70:", "            num = 3", "        elif 70 <= 147 < 150:", "            num = 2", "        else:", "            num = 1", "        for i in range(num):", "            m = np.concatenate((m, mat), axis=1)", ">           output = submatrix_with_n_numbers(m)"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q58/command_results_2/Folder_52/tests58_52_2.py", "lineno": 56, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    return np.sum(np.sum(np.sum(np.sum(matrix == 147, axis=1), axis=2), axis=3) == 147)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q58/command_results_2/Folder_52/generated_answer.py", "lineno": 4, "message": "in submatrix_with_n_numbers"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return _wrapreduction(a, np.add, 'sum', axis, dtype, out, keepdims=keepdims,"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/fromnumeric.py", "lineno": 2313, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return reduction(axis=axis, out=out, **passkwargs)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/fromnumeric.py", "lineno": 86, "message": "in _wrapreduction"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return N.ndarray.sum(self, axis, dtype, out, keepdims=True)._collapse(axis)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/matrixlib/defmatrix.py", "lineno": 320, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def _sum(a, axis=None, dtype=None, out=None, keepdims=False,", "             initial=_NoValue, where=True):", ">       return umr_sum(a, axis, dtype, out, keepdims, initial, where)", "E       numpy.exceptions.AxisError: axis 2 is out of bounds for array of dimension 2"], "reprfuncargs": {"args": [["a", "matrix([[0]])"], ["axis", "2"], ["dtype", "None"], ["out", "None"], ["keepdims", "True"], ["initial", "<no value>"], ["where", "True"]]}, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "AxisError"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_horizontal_matrix():", "        mat = [[random.randint(-10, 10) for _ in range(147)]]", "        mat = np.asmatrix(mat)", "        m = np.concatenate((mat, mat), axis=1)", "        if 147 < 30:", "            num = 4", "        elif 30 <= 147 < 70:", "            num = 3", "        elif 70 <= 147 < 150:", "            num = 2", "        else:", "            num = 1", "        for i in range(num):", "            m = np.concatenate((m, mat), axis=1)", ">           output = submatrix_with_n_numbers(m)"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q58/command_results_2/Folder_52/tests58_52_2.py", "lineno": 56, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    return np.sum(np.sum(np.sum(np.sum(matrix == 147, axis=1), axis=2), axis=3) == 147)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q58/command_results_2/Folder_52/generated_answer.py", "lineno": 4, "message": "in submatrix_with_n_numbers"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return _wrapreduction(a, np.add, 'sum', axis, dtype, out, keepdims=keepdims,"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/fromnumeric.py", "lineno": 2313, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return reduction(axis=axis, out=out, **passkwargs)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/fromnumeric.py", "lineno": 86, "message": "in _wrapreduction"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return N.ndarray.sum(self, axis, dtype, out, keepdims=True)._collapse(axis)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/matrixlib/defmatrix.py", "lineno": 320, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def _sum(a, axis=None, dtype=None, out=None, keepdims=False,", "             initial=_NoValue, where=True):", ">       return umr_sum(a, axis, dtype, out, keepdims, initial, where)", "E       numpy.exceptions.AxisError: axis 2 is out of bounds for array of dimension 2"], "reprfuncargs": {"args": [["a", "matrix([[0]])"], ["axis", "2"], ["dtype", "None"], ["out", "None"], ["keepdims", "True"], ["initial", "<no value>"], ["where", "True"]]}, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "AxisError"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "numpy.exceptions.AxisError: axis 2 is out of bounds for array of dimension 2"}, null]]}, "when": "call", "user_properties": [], "sections": [], "duration": 0.00035802600905299187, "start": 1731152876.1251674, "stop": 1731152876.125526, "extras": [], "$report_type": "TestReport", "item_index": 2, "worker_id": "gw2", "testrun_uid": "3d3ace11be9e4282adfa52696af4d5e4", "node": "<WorkerController gw2>"}
{"nodeid": "tests58_52_2.py::test_horizontal_matrix", "location": ["tests58_52_2.py", 40, "test_horizontal_matrix"], "keywords": {"test_horizontal_matrix": 1, "__wrapped__": 1, "tests58_52_2.py": 1, "Folder_52": 1}, "outcome": "passed", "longrepr": null, "when": "teardown", "user_properties": [], "sections": [], "duration": 0.0001594738569110632, "start": 1731152876.1627648, "stop": 1731152876.1629252, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw2", "testrun_uid": "3d3ace11be9e4282adfa52696af4d5e4", "node": "<WorkerController gw2>"}
{"nodeid": "tests58_52_2.py::test_matrix_of_one_integer", "location": ["tests58_52_2.py", 18, "test_matrix_of_one_integer"], "keywords": {"test_matrix_of_one_integer": 1, "__wrapped__": 1, "tests58_52_2.py": 1, "Folder_52": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "numpy.exceptions.AxisError: axis 2 is out of bounds for array of dimension 2"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_one_integer():", "        input_matrix = [[147]]", "        input_matrix = np.asmatrix(input_matrix)", "        if 147 == 1:", "            assert submatrix_with_n_numbers(input_matrix) == 1", "        else:", ">           assert not submatrix_with_n_numbers(input_matrix)"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q58/command_results_2/Folder_52/tests58_52_2.py", "lineno": 26, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    return np.sum(np.sum(np.sum(np.sum(matrix == 147, axis=1), axis=2), axis=3) == 147)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q58/command_results_2/Folder_52/generated_answer.py", "lineno": 4, "message": "in submatrix_with_n_numbers"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return _wrapreduction(a, np.add, 'sum', axis, dtype, out, keepdims=keepdims,"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/fromnumeric.py", "lineno": 2313, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return reduction(axis=axis, out=out, **passkwargs)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/fromnumeric.py", "lineno": 86, "message": "in _wrapreduction"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return N.ndarray.sum(self, axis, dtype, out, keepdims=True)._collapse(axis)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/matrixlib/defmatrix.py", "lineno": 320, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def _sum(a, axis=None, dtype=None, out=None, keepdims=False,", "             initial=_NoValue, where=True):", ">       return umr_sum(a, axis, dtype, out, keepdims, initial, where)", "E       numpy.exceptions.AxisError: axis 2 is out of bounds for array of dimension 2"], "reprfuncargs": {"args": [["a", "matrix([[1]])"], ["axis", "2"], ["dtype", "None"], ["out", "None"], ["keepdims", "True"], ["initial", "<no value>"], ["where", "True"]]}, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "AxisError"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_one_integer():", "        input_matrix = [[147]]", "        input_matrix = np.asmatrix(input_matrix)", "        if 147 == 1:", "            assert submatrix_with_n_numbers(input_matrix) == 1", "        else:", ">           assert not submatrix_with_n_numbers(input_matrix)"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q58/command_results_2/Folder_52/tests58_52_2.py", "lineno": 26, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    return np.sum(np.sum(np.sum(np.sum(matrix == 147, axis=1), axis=2), axis=3) == 147)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q58/command_results_2/Folder_52/generated_answer.py", "lineno": 4, "message": "in submatrix_with_n_numbers"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return _wrapreduction(a, np.add, 'sum', axis, dtype, out, keepdims=keepdims,"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/fromnumeric.py", "lineno": 2313, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return reduction(axis=axis, out=out, **passkwargs)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/fromnumeric.py", "lineno": 86, "message": "in _wrapreduction"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return N.ndarray.sum(self, axis, dtype, out, keepdims=True)._collapse(axis)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/matrixlib/defmatrix.py", "lineno": 320, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def _sum(a, axis=None, dtype=None, out=None, keepdims=False,", "             initial=_NoValue, where=True):", ">       return umr_sum(a, axis, dtype, out, keepdims, initial, where)", "E       numpy.exceptions.AxisError: axis 2 is out of bounds for array of dimension 2"], "reprfuncargs": {"args": [["a", "matrix([[1]])"], ["axis", "2"], ["dtype", "None"], ["out", "None"], ["keepdims", "True"], ["initial", "<no value>"], ["where", "True"]]}, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "AxisError"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "numpy.exceptions.AxisError: axis 2 is out of bounds for array of dimension 2"}, null]]}, "when": "call", "user_properties": [], "sections": [], "duration": 0.0002586699556559324, "start": 1731152876.1266649, "stop": 1731152876.126924, "extras": [], "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "3d3ace11be9e4282adfa52696af4d5e4", "node": "<WorkerController gw0>"}
{"nodeid": "tests58_52_2.py::test_matrix_of_one_integer", "location": ["tests58_52_2.py", 18, "test_matrix_of_one_integer"], "keywords": {"test_matrix_of_one_integer": 1, "__wrapped__": 1, "tests58_52_2.py": 1, "Folder_52": 1}, "outcome": "passed", "longrepr": null, "when": "teardown", "user_properties": [], "sections": [], "duration": 0.00014908402226865292, "start": 1731152876.1851072, "stop": 1731152876.185257, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "3d3ace11be9e4282adfa52696af4d5e4", "node": "<WorkerController gw0>"}
{"nodeid": "tests58_52_2.py::test_matrix_of_having_147_integers", "location": ["tests58_52_2.py", 28, "test_matrix_of_having_147_integers"], "keywords": {"test_matrix_of_having_147_integers": 1, "__wrapped__": 1, "tests58_52_2.py": 1, "Folder_52": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "numpy.exceptions.AxisError: axis 2 is out of bounds for array of dimension 2"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_having_147_integers():", "        if 147 % 2 == 0:", "            input_matrix = [[random.randint(-100, 100) for _ in range(2)] for _ in range(147 // 2)]", "            input_matrix = np.asmatrix(input_matrix)", "        else:", "            input_matrix = [[random.randint(-100, 100) for _ in range(147)]]", "            input_matrix = np.asmatrix(input_matrix)", "    ", ">       assert submatrix_with_n_numbers(input_matrix) == 1"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q58/command_results_2/Folder_52/tests58_52_2.py", "lineno": 38, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    return np.sum(np.sum(np.sum(np.sum(matrix == 147, axis=1), axis=2), axis=3) == 147)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q58/command_results_2/Folder_52/generated_answer.py", "lineno": 4, "message": "in submatrix_with_n_numbers"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return _wrapreduction(a, np.add, 'sum', axis, dtype, out, keepdims=keepdims,"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/fromnumeric.py", "lineno": 2313, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return reduction(axis=axis, out=out, **passkwargs)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/fromnumeric.py", "lineno": 86, "message": "in _wrapreduction"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return N.ndarray.sum(self, axis, dtype, out, keepdims=True)._collapse(axis)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/matrixlib/defmatrix.py", "lineno": 320, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def _sum(a, axis=None, dtype=None, out=None, keepdims=False,", "             initial=_NoValue, where=True):", ">       return umr_sum(a, axis, dtype, out, keepdims, initial, where)", "E       numpy.exceptions.AxisError: axis 2 is out of bounds for array of dimension 2"], "reprfuncargs": {"args": [["a", "matrix([[0]])"], ["axis", "2"], ["dtype", "None"], ["out", "None"], ["keepdims", "True"], ["initial", "<no value>"], ["where", "True"]]}, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "AxisError"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_having_147_integers():", "        if 147 % 2 == 0:", "            input_matrix = [[random.randint(-100, 100) for _ in range(2)] for _ in range(147 // 2)]", "            input_matrix = np.asmatrix(input_matrix)", "        else:", "            input_matrix = [[random.randint(-100, 100) for _ in range(147)]]", "            input_matrix = np.asmatrix(input_matrix)", "    ", ">       assert submatrix_with_n_numbers(input_matrix) == 1"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q58/command_results_2/Folder_52/tests58_52_2.py", "lineno": 38, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    return np.sum(np.sum(np.sum(np.sum(matrix == 147, axis=1), axis=2), axis=3) == 147)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q58/command_results_2/Folder_52/generated_answer.py", "lineno": 4, "message": "in submatrix_with_n_numbers"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return _wrapreduction(a, np.add, 'sum', axis, dtype, out, keepdims=keepdims,"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/fromnumeric.py", "lineno": 2313, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return reduction(axis=axis, out=out, **passkwargs)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/fromnumeric.py", "lineno": 86, "message": "in _wrapreduction"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return N.ndarray.sum(self, axis, dtype, out, keepdims=True)._collapse(axis)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/matrixlib/defmatrix.py", "lineno": 320, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def _sum(a, axis=None, dtype=None, out=None, keepdims=False,", "             initial=_NoValue, where=True):", ">       return umr_sum(a, axis, dtype, out, keepdims, initial, where)", "E       numpy.exceptions.AxisError: axis 2 is out of bounds for array of dimension 2"], "reprfuncargs": {"args": [["a", "matrix([[0]])"], ["axis", "2"], ["dtype", "None"], ["out", "None"], ["keepdims", "True"], ["initial", "<no value>"], ["where", "True"]]}, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "AxisError"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "numpy.exceptions.AxisError: axis 2 is out of bounds for array of dimension 2"}, null]]}, "when": "call", "user_properties": [], "sections": [], "duration": 0.000346075976267457, "start": 1731152876.130649, "stop": 1731152876.1309958, "extras": [], "$report_type": "TestReport", "item_index": 1, "worker_id": "gw1", "testrun_uid": "3d3ace11be9e4282adfa52696af4d5e4", "node": "<WorkerController gw1>"}
{"nodeid": "tests58_52_2.py::test_matrix_of_having_147_integers", "location": ["tests58_52_2.py", 28, "test_matrix_of_having_147_integers"], "keywords": {"test_matrix_of_having_147_integers": 1, "__wrapped__": 1, "tests58_52_2.py": 1, "Folder_52": 1}, "outcome": "passed", "longrepr": null, "when": "teardown", "user_properties": [], "sections": [], "duration": 0.00011992291547358036, "start": 1731152876.192853, "stop": 1731152876.1929739, "$report_type": "TestReport", "item_index": 1, "worker_id": "gw1", "testrun_uid": "3d3ace11be9e4282adfa52696af4d5e4", "node": "<WorkerController gw1>"}
{"exitstatus": 2, "$report_type": "SessionFinish"}
